docs/source/index.rst

```diff
@@ -4,28 +4,47 @@ Welcome to OpenDCS's documentation!
 **OpenDCS** suite of software for collecting, decodes, storing, and 
 manipulating time series data.
 
-Check out the :doc:`quick-start` section for how to get started.
+Check out the :doc:`legacy-quick-start` section for how to get started.
 
 .. note::
 
    This project is under active development.
 
-Contents
---------
+Documentation Outline
+---------------------
+OpenDCS documentation is currently divided into three sections
 
 .. toctree::
    :maxdepth: 1
+   :caption: Getting Started
    
-
-   Quick Start <./quick-start.rst>
-   Install Guide <./install-guide.rst>
-   LRGS Guide <./lrgs-userguide.rst>
-   Computation Guide <./cp-userguide.rst>
-   DECODES Guide <./decoding-guide.rst>
-   Routing & Scheduling Guide <./routesched-guide.rst>
-   Snotel Addendum <./snotel.rst>
-   TimeSeriesDatabase <./tsdb.rst>
-   Alarms <./alarms.rst>
-   CWMS Interface <./cwms-interface.rst>
-   Developer Documentation <./dev-docs.rst>
+   Software Set-Up <./start-software.rst>
+   Computations - Intro <./start-computations.rst>
+   Routing Spec - Intro<./start-routingspec.rst>
+   Screening - Intro <./start-screening.rst>
+   
+.. toctree::
+   :maxdepth: 1
+   :caption: Resources
+   
+   Algorithms <./resources-algorithms.rst>   
+   Computations <./resources-computations.rst>
+   DECODING <./resources-decoding.rst>
+   Screening Editor <./resources-screening.rst>
+   
+.. toctree::
+   :maxdepth: 1
+   :caption: Legacy Documentation
+   
+   Quick Start <./legacy-quick-start.rst>
+   Install Guide <./legacy-install-guide.rst>
+   LRGS Guide <./legacy-lrgs-userguide.rst>
+   Computation Guide <./legacy-cp-userguide.rst>
+   DECODES Guide <./legacy-decoding-guide.rst>
+   Routing & Scheduling Guide <./legacy-routesched-guide.rst>
+   Snotel Addendum <./legacy-snotel.rst>
+   TimeSeriesDatabase <./legacy-tsdb.rst>
+   Alarms <./legacy-alarms.rst>
+   CWMS Interface <./legacy-cwms-interface.rst>
+   Developer Documentation <./legacy-dev-docs.rst>
```

Comment:
The dev documentation isn't legacy. Move it to it's own root level, just remove "legacy documentation."

Reply:
I changed it to "Developer Documentation" and included 3 bullets at the top of the index.